test/snapd_test.dart

```diff
@@ -1177,8 +1177,8 @@ void main() {
     expect(info.kernelVersion, equals('5.11.0'));
     expect(info.managed, isTrue);
     expect(info.onClassic, isTrue);
-    expect(info.refreshLast, equals(DateTime.utc(2022, 5, 28, 20, 10)));
-    expect(info.refreshNext, equals(DateTime.utc(2022, 5, 29, 1, 18)));
+    expect(info.refresh.last, equals(DateTime.utc(2022, 5, 28, 20, 10)));
+    expect(info.refresh.next, equals(DateTime.utc(2022, 5, 29, 1, 18)));
```

Comment:
Can we do this without changing the API, perhaps make properties that provide the old names?

Reply:
There are getters for backward compatibility but I marked them deprecated so I had to update the test to avoid lint failures.

https://github.com/canonical/snapd.dart/blob/d2ab9b0cf8491019514defc62e555ba20d532eb6/lib/src/snapd_client.dart#L572-L576